panel/.env.example

```
@@ -0,0 +1,13 @@
+NODE_ENV=development
+HOST=127.0.0.1
+PORT=3000
+PUBLIC_URL=http://localhost:3000
+DATABASE_URL=postgres://zomboid_panel:change-me@localhost:5432/zomboid_panel
+SESSION_SECRET=replace-with-a-long-random-secret
+AGENT_ENROLLMENT_TOKEN=replace-with-a-long-random-secret
+AGENT_STALE_SECONDS=60
+# Local development only; production ignores both flags.
+PZ_FAKE_AGENT=1
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PZ_FAKE_AGENT key should go before the SESSION_SECRET key

(UnorderedKey)